package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
```

Comment:
## Your Homepage
![image](https://user-images.githubusercontent.com/93039359/225508031-be661650-dfbc-44dd-ab6f-b79bd43e17b3.png)

## Design Homepage
![image](https://user-images.githubusercontent.com/93039359/225508248-9514a73c-6845-4e30-b478-85b75aa6d408.png)

**[OPTIONAL]**

- [ ] Great implementation of the design but i guess the design you have implemented for your homepage was actually for your details page as you can see in the second picture above how your homepage should look like 😊
- [ ] It would be nice if you add those missing things I have highlighted with blue marker in the Design image above 👍

Kindly make both of these changes to make your project look professional and follow all the design requirements 😊